README.md

```diff
@@ -1,2 +1,23 @@
 # skycoin-services
+
 skycon-services
+
+## dmsg daemon
+
+This daemon will create `.dmsg-uuid` file, which contains your unique UUID.  
```

Comment:
It would help if you could summarise the purpose of this utility in 2-3 sentences as well.